tests/std/tests/P2442R1_views_chunk/test.cpp

```diff
@@ -35,6 +35,9 @@ constexpr bool test_one(Rng&& rng, Expected&& expected) {
     STATIC_ASSERT(forward_range<R> == forward_range<Rng>);
     STATIC_ASSERT(bidirectional_range<R> == bidirectional_range<Rng>);
     STATIC_ASSERT(random_access_range<R> == random_access_range<Rng>);
+    // Validate non-default-initializability
+    STATIC_ASSERT(!is_default_constructible_v<R>);
+    STATIC_ASSERT(!default_initializable<R>);

```

Comment:
No change requested: `test_one()` is called for both input-only and forward ranges, so this covers the LWG issue's modifications to both the primary template and the specialization for forward ranges.